openedx/stanford/djangoapps/auth_lagunita/management/commands/backfill_auth_lagunita_extrainfo.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+Backfill extra registration info for existing users
+"""
+from __future__ import unicode_literals
+import logging
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.db import transaction, IntegrityError
+
+from ...models import Info
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Migrate data from the legacy system
+    """
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-i',
+            '--insert',
+            action='store_true',
+            default=False,
+            help='Insert new records into the database',
+        )
+
+    def handle(self, *args, **kwargs):
+        users = User.objects.all()
+        for user in users:
+            try:
+                info_old = Info.objects.get(user=user)
+            except Info.DoesNotExist:
+                pass
+            else:
+                log.info(
+                    "Unexpectedly found existing Info for user: %(email)s",
+                    {
+                        'email': user.email,
+                    }
+                )
+                continue
+            info_new = Info()
+            info_new.id = user.id
```

Comment:
I'm a bit lost why you're inserting the user.id into the primary key. Should it not be a sequence?

Reply:
The more I think about it, the more I think I borrowed this idiom from CME registration.
I think the idea _then_ was to reuse the _old_ CME registration entry ID for the corresponding _new_ CME registration ID.

Assuming that's the case, this would be necessary.

Reply:
Sorry, meant to say "would be _un_ necessary.

This has been removed.